src/utils/raw.rs

```diff
@@ -311,7 +311,8 @@ pub fn remove_dir(path: &Path) -> io::Result<()> {
 
 pub fn copy_dir(src: &Path, dest: &Path) -> io::Result<()> {
     fs::create_dir(dest)?;
-    for entry in src.read_dir()? {
+    let diriter = src.read_dir()?;
+    for entry in diriter {
```

Comment:
While I think this is a perfectly acceptable neatness/comprehensiblity update, it seems like a bogon in this PR.  Don't mind if you want to retain it; but if you want to clean it up later then I'd also be happy.